app/views/spree/admin/shared/_configuration_menu.html.haml

```diff
@@ -0,0 +1,30 @@
+- content_for :sidebar_title do
+  = Spree.t(:configurations)
```

Comment:
I didnt want to touch translations in this PR. We should probably remove all Spree.t entries (there are more) from ofn's code and move everything we can to our namespaces.